managers/asset-operations-manager.js

```diff
@@ -345,43 +345,57 @@ class AssetOperationsManager {
 
         const { tokenId } = resolveUAL(UAL);
 
-        let publicAssertionId;
-        if(state === ASSET_STATES.LATEST) {
+        let publicAssertionId,
+            stateFinalized = false;
+        if (state === ASSET_STATES.LATEST) {
             const unfinalizedState = await this.blockchainService.getUnfinalizedState(
                 tokenId,
                 blockchain,
             );
 
             if (unfinalizedState != null && unfinalizedState !== ZeroHash) {
                 publicAssertionId = unfinalizedState;
+                stateFinalized = false;
+            } else {
+                stateFinalized = true;
```

Comment:
We can also avoid this, my bad. We're checking it later on line 371

Reply:
removed the else, you're right